tools/IceRpc.Slice.Tools/SliceFile.ProjectProperties.xaml

```diff
@@ -33,6 +33,15 @@
         </StringProperty.DataSource>
     </StringProperty>
 
+    <EnumProperty Name="Rpc"
+                  DisplayName="Rpc Provider"
```

Comment:
Same here.

Also, do we need in both SliceFile.FileProperties.xaml and SliceFile.ProjectProperties.xaml?

Reply:
One is per file, and the other is per project. This is to allow setting things from Visual Studio UI